eng/tools/GenerateFiles/Directory.Build.targets.in

```diff
@@ -26,19 +29,27 @@
       LatestRuntimeFrameworkVersion="${MicrosoftNETCoreAppRuntimeVersion}"
```

Comment:
@ericstj and @Pilchie question that's not directly related to this PR:

We set up this `@(KnownFrameworkReference)` so that we always compile against the latest available targeting pack and shared framework from dotnet/runtime. That's probably fine for release/5.0 unless we service Microsoft.NETCore.App.Ref. But, if we do, I think this approach will set a new minimum shared framework version e.g. 5.0.2 in any package where we ship generated `*.deps.json` or `*.runtimeconfig.json` files. Is that something we should worry about❔